app/assets/javascripts/mapping.js

```diff
@@ -69,16 +28,24 @@ var MRConfig = (function () {
 (function() {
     var map;
     $(document).ready(function() {
-        var road_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.png', {
```

Comment:
We could clean this up a little by using [these shortcuts](https://gist.github.com/mourner/1804938). No idea if these ever made it into Leaflet core.